data/pass_sport_campaign_export.py

```diff
@@ -69,17 +52,23 @@
 }
 
 # df_formatted = df_unwrapped_alloc.rename(columns=column_mapping) 
-df_unwrapped_alloc.columns = df_unwrapped_alloc.columns.to_series().replace(column_mapping)
+df_main.columns = df_main.columns.to_series().replace(column_mapping)
+
+
+# In[23]:
+
+
+df_campaign = df_main[['beneficiaire_prenom', 'beneficiaire_nom', 'beneficiaire_genre', 'beneficiaire_date_naissance', 'code']]
 
 
-# In[ ]:
+# In[24]:
 
 
-df_unwrapped_alloc = df_unwrapped_alloc[['allocataire_email', 'allocataire_qualite', 'allocataire_nom', 'allocataire_prenom',
-'beneficiaire_prenom', 'beneficiaire_nom', 'beneficiaire_genre', 'beneficiaire_date_naissance', 'code']]
+df_campaign['beneficiaire_date_naissance'] = pd.to_datetime(df_campaign['beneficiaire_date_naissance'].apply(lambda v: v[:10]), format='%Y-%m-%d')
+df_campaign['beneficiaire_date_naissance'] = df_campaign['beneficiaire_date_naissance'].dt.strftime('%d/%m/%Y')
```

Comment:
Le format d'insertion dans la bdd doit être YYYY-MM-DD si je ne dis pas de bêtises

```suggestion
df_campaign['beneficiaire_date_naissance'] = df_campaign['beneficiaire_date_naissance'].dt.strftime('%Y-%m-%d')
```

Reply:
Ce notebook c'est pour l'export pour le presta de mail. Il ont demandé le format dd/mm/YYYY

Reply:
Ah bien vu, j'avais pas fais gaffe x)

Reply:
J'ai rename les notebooks pour que ça soit plus clair